scrapers/IFeelMyself.py

```diff
@@ -10,17 +10,20 @@ def readJSONInput():
     input = sys.stdin.read()
     return json.loads(input)
 
-def extract_info(table):
-    description=table.find(class_= ["blog_wide_new_text","entryBlurb"]).get_text(strip=True).replace("\x92","'")
+def extract_info(table,cover_url=None):
+    if table.find(class_= ["blog_wide_new_text","entryBlurb"]):
+        description=table.find(class_= ["blog_wide_new_text","entryBlurb"]).get_text(strip=True).replace("\x92","'")
+    description = None
```

Comment:
this makes the `if` above irrelevant, shouldnt it go above the `if table.find` instead?